hat/assets/js/apps/Iaso/domains/entities/entityTypes/hooks/requests/forms.ts

```diff
@@ -28,11 +28,15 @@ export const useGetForm = (
 
 export const useGetForms = (
     enabled: boolean,
+    fields?: string[] | undefined,
 ): UseQueryResult<Form[], Error> => {
+    let url = '/api/forms/?fields=id,name,latest_form_version';
+    if (fields) {
+        url += `,${fields.join(',')}`;
+    }
```

Comment:
nitpick but I'm not a big fan of this pattern
```suggestion
    const apiUrl = '/api/forms/?fields=id,name,latest_form_version'; // I would even hoist that outside of the hook
    const url = fields ? `${apiUrl},${fields.join(',')}` : apiUrl;
```

Reply:
Not sure I find it more readable, but changed :+1: